Source/Utils/WarpXUtil.H

```diff
@@ -24,6 +24,8 @@ void ReadBoostedFrameParameters(amrex::Real& gamma_boost, amrex::Real& beta_boos
 
 void ConvertLabParamsToBoost();
 
+void CheckGriddingForRZSpectral();
```

Comment:
```suggestion
/** When using the RZ spectral solver, the Hankel transform cannot be
 * divided among multiple blocks. Each block must extend over the
 * entire radial extent. The grid can be divided up along z, but the number of blocks
 * must be >= the number of processors. This functions checks that these conditions
 * are satisfied. */
void CheckGriddingForRZSpectral();
```

Reply:
My preference is that the detailed description be with the function and not its header, so that you can read what it is doing while looking at its source. How about a short description in the header file, like
`// Ensures that the blocks are setup correctly for the RZ spectral solver`

Reply:
Absolutely, sounds good to me (except You'll need syntax `/** description */` instead of `// description`)

Reply:
Change made